CIP-0068/README.md

```diff
@@ -115,9 +116,15 @@ metadata =
     ? mediaType : bounded_bytes, ; UTF-8
     ? description : bounded_bytes, ; UTF-8
     ? files : [* files_details]
+    ; ... Additional properties are allowed
```

Comment:
Not sure to follow this line 🤔 ? Is this in reference to the new `extra` ? In which case, additional properties aren't really on the `metadata` record, right?

Reply:
This comment is just to clarify that additional proprieties are allowed within the `metadata` field.

Reply:
@KtorZ it's not really related to the extra field. Just thought I sneak it into the PR, wanted to make it more explicit that additional properties are allowed.